src/hooks/useFetch.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect } from "react";
+
+function useFetch(url) {
+  const [apiDB, setApiDB] = useState();
+  const [isLoading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    fetch(url)
+      .then((res) => res.json())
+      .then((data) => {
+        setApiDB(data);
+        setLoading(false);
+        console.log("DATA FETCHED: useFetch");
+      });
```

Comment:
Great! so here you will add a `catch` statment, and setLoading to false and Error to true 🎯